pavelib/assets.py

```diff
@@ -292,6 +301,27 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
```

Comment:
@feanil pointed me at this Ansible script which is different for production than for sandboxes:

https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/tasks/service_variant_config.yml

We'll have to look at this together tomorrow to see whether this is going to be an issue. There is a comment that says "Gather assets using paver if possible" so I'm hopeful that this will "just work".